__tests__/app.test.js

```diff
@@ -333,4 +354,25 @@ describe('Error handling tests', () => {
             })
         })
     })
+    describe('GET `/api/users` errors', () => {
```

Comment:
This issue, where the tables are missing is more like a server side issue, rather than a client side one.